ironfish/src/rpc/routes/chain/getTransactionStream.ts

```diff
@@ -147,20 +211,30 @@ router.register<typeof GetTransactionStreamRequestSchema, GetTransactionStreamRe
       })
     }
 
+    const validatedAssetValue = async (assetIdentifier: Buffer): Promise<AssetsValue> => {
+      const assetValue = await node.chain.assets.get(assetIdentifier)
+      if (!assetValue) {
```

Comment:
you should use node.chain.getAssetById and not this store because it handles the default asset which isnt in the DB yet.